<launch>
    <node pkg="xr1controllerol" type="actuator_bridge" name="actuator_bridge" output="screen" respawn="false">
         <param name="param_file" value="EmeraldEndive.xr1para"/>
         <param name="tf_switch" value="1" type="int"/>
    </node>
</launch>